plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java

```diff
@@ -52,11 +52,14 @@ public Boolean checkingHB() {
         cmd.add("-t", String.valueOf(String.valueOf(System.currentTimeMillis() / 1000)));
         cmd.add("-d", String.valueOf(suspectTimeInSeconds));
         OutputInterpreter.OneLineParser parser = new OutputInterpreter.OneLineParser();
+
```

Comment:
@GutoVeronezi may be you can also improve this method name,  `checkingHB()` => _checkingHeartBeat()_

Reply:
**+1** HB is not really telling much